### DISCUSSION TITLE
Update CDU error

### DISCUSSION THREAD

**Discussion by LFrank2021** [opendcim/openDCIM#1506] [Q&A] [answered]
Hi,

I am trying to update a CDU with more power ports (in my case 36) and receive this error:
[Wed Jan 31 16:21:59.400257 2024] [php:error] [pid 32274] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '986-1' for key 'PRIMARY' in /srv/openDCIM/openDCIM-23.04/classes/PowerPorts.class.php:106\nStack trace:\n#0 /srv/openDCIM/openDCIM-23.04/classes/PowerPorts.class.php(106): PDO->query()\n#1 /srv/openDCIM/openDCIM-23.04/classes/PowerPorts.class.php(152): PowerPorts->createPort()\n#2 /srv/openDCIM/openDCIM-23.04/classes/Device.class.php(899): PowerPorts::createPorts()\n#3 /srv/openDCIM/openDCIM-23.04/devices.php(723): Device->UpdateDevice()\n#4 {main}\n  thrown in /srv/openDCIM/openDCIM-23.04/classes/PowerPorts.class.php on line 106, referer: https://mydcim/devices.php?DeviceID=986

Same happens on demo.opendcim.org

Kind regards
Frank

**Comment by wilpig** [comment, marked as answer]
closed with issue #1507
